modules/administration-guide/pages/installing-che-on-minikube.adoc

```diff
@@ -1,21 +1,51 @@
-:_content-type: ASSEMBLY
+:_content-type: PROCEDURE
 :navtitle: Installing {prod-short} on Minikube
 :keywords: overview, installing-che-on-minikube
 :page-aliases: installation-guide:installing-che-on-minikube, overview:installing-che-on-minikube
 
-:parent-context-of-installing-che-on-minikube: {context}
 
 [id="installing-{prod-id-short}-on-minikube_{context}"]
 = Installing {prod-short} on Minikube
 
-:context: installing-{prod-id-short}-on-minikube
-
 This article explains how to create a single-node {kubernetes} cluster with Minikube to deploy {prod-short}.
 
 WARNING: Remember that single-node {kubernetes} clusters are suited only for testing or development. Do *NOT* use such clusters to run {prod-short} for organizations or developer teams.
 
-include::partial$proc_using-minikube-to-set-up-kubernetes.adoc[leveloffset=+1]
+.Prerequisites
+
+* `{orch-cli}`. See link:https://kubernetes.io/docs/tasks/tools/#kubectl/[Installing and Setting Up {orch-cli} ].
```

Comment:
Make consistent with the same prerequisite in other files.

Reply:
Done